### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by muellerzr**
Adds a pr template with the relevant area delegations

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
LGTM

**Review by SunMarc — APPROVED** [review]
LGTM !

**Review by BenjaminBossan — APPROVED** [review]
Looks good, nothing to add from my side.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
+members/contributors who may be interested in your PR.
+
+<!-- Your PR will be replied to more quickly if you can figure out the right person to tag with @
+
+ If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
+
+- Big modeling: @SunMarc
+- Fully-Sharded Data Parallism: @pacsun100
```
Comment (statelesshz) [inline .github/PULL_REQUEST_TEMPLATE.md:40]: Maybe it’s pacman100?

Comment (muellerzr) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nice catch!

**Review by pacman100 — APPROVED** [review]
Thank you, LGTM! 🤗

**Pull request merged**